### ISSUE TITLE
consider specifying the expected mappings when `aria-readonly` is unspecified on both `grid` and `gridcell`

### ISSUE THREAD

**Issue by mfairchild365** [w3c/core-aam#71]
After doing some research, I'm a little confused about what is expected when `aria-readonly` is unspecified on both `grid` and `gridcell`.

From [ARIA 1.1's `gridcell` definition
](https://www.w3.org/TR/wai-aria-1.1/#gridcell)

> An unspecified value for aria-readonly does not imply that a grid or a gridcell contains editable content. For example, if a grid presents a collection of elements that are not editable, such as a collection of link elements representing dates in a datepicker, it is not necessary for the author to specify a value for aria-readonly.

I read this to mean that if both `grid` and `gridcell` do not have `aria-readonly` defined by the author, all descendant `gridcell` roles should be readonly.

Core AAM appears to only specify what happens when `aria-readonly` is undefined on the `gridcell` role, not the `grid` role.

> The gridcell MUST inherit any author-provided value for aria-readonly from the containing grid or treegrid. Expose the inherited value on the gridcell as described for aria-readonly="true" and aria-readonly="false".

This language seems to suggest that only "author-provided" values of `aria-readonly` should propagate to descendant `gridcell` roles. Thus, an unspecified value on the `grid` should not propagate.

If my understanding is correct, these two definitions do not appear to be in harmony. It may help to clarify the expected mappings when `aria-readonly` is unspecified on both roles.
